smart-app-banner.css

```diff
@@ -10,7 +10,7 @@
 .smartbanner {
 	position: absolute;
 	left: 0;
-	top: 0;
+	top: -80;
```

Comment:
You are right @dfcreative . I tried auto, and it works. I'll create a different merge request.